openchallenges/data_integration_lambda.py

```diff
@@ -0,0 +1,38 @@
+from aws_cdk import aws_iam as iam
+from aws_cdk import aws_lambda as _lambda
+from constructs import Construct
+
+
+class DataIntegrationLambda(Construct):
+
+    def __init__(self, scope: Construct, id: str) -> None:
+        super().__init__(scope, id)
+
+        self.lambda_role = self._build_lambda_role()
+        self.lambda_function = self._build_lambda_function(self.lambda_role)
```

Comment:
Could you add in a docstring that covers the input arguments?

Reply:
Done